integration-tests/requirements.txt

```diff
@@ -4,14 +4,19 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
+# To install pip-compile, run:
+#    pip install pip-tools
+#
 behave==1.2.5
 docker-py==1.10.6
 docker-pycreds==0.2.1     # via docker-py
+enum34==1.1.6             # via parse-type
```

Comment:
This was added because you used python2 version of pip-compile. If you installed pip-compile with `pip3 install pip-tools` this would be not added, because `enum34` is a back-port of python3.4's enum to python2.